app/_hub/kong-inc/openid-connect/overview/_index.md

```diff
@@ -1771,14 +1771,14 @@ The certificate-bound access tokens are supported by the following:
 
 The following example shows how to enable this feature for the JWT Access Token Authentication method. Similar steps can be followed for the other methods.
 
-1. Configure {{site.base_gateway}} to use mTLS client certificate authentication. You can do this by configuring the [TLS Handshake Modifier plugin](/hub/kong-inc/tls-handshake-modifier/) or the [Mutual TLS Authentication plugin](/hub/kong-inc/mtls-auth/):
+1. Configure {{site.base_gateway}} to use mTLS client certificate authentication. This can be achieved by configuring the [TLS Handshake Modifier plugin](/hub/kong-inc/tls-handshake-modifier/) or the [Mutual TLS Authentication plugin](/hub/kong-inc/mtls-auth/):
```

Comment:
The mtls plugins are mentioned here but the dependency of the pop feature on either of mTLS plugins is not explicitly called out. It be a good option to include that as a 2nd Note.

Reply:
agreed. BTW, there is an additional note about this that is printed during schema validation: https://github.com/Kong/kong-ee/pull/6446/files#diff-b4a07d5d94cf8e3a85f63772fba4ebdc58b0f2516af91e384da67cea5b62477eR162